samples/dotnet/clientcreds/Lenus.Samples.ClientCredentialsFlow/src/wwwroot/js/site.js

```diff
@@ -0,0 +1,4 @@
+﻿// Please see documentation at https://docs.microsoft.com/aspnet/core/client-side/bundling-and-minification
+// for details on configuring this project to bundle and minify static web assets.
+
+// Write your JavaScript code.
```

Comment:
Is this file required? Seems empty

Reply:
This is part of the default project templates, if removed dont forget to also update the _Layout.cshtml